src/main/java/net/media/openrtb3/Seatbid.java

```diff
@@ -52,11 +52,11 @@ public void set_package(Integer _package) {
     this._package = _package;
   }
 
-  public @NotNull @Valid Collection<Bid> getBid() {
+  public @NotEmpty @Valid Collection<Bid> getBid() {
     return this.bid;
   }
 
-  public void setBid(@NotNull @Valid Collection<Bid> bid) {
+  public void setBid(@NotEmpty @Valid Collection<Bid> bid) {
```

Comment:
Check this in openRtb doc if it allows empty bid array. As bidders can send this.